docs/src/stories/components/ToggleSwitch/ToggleSwitch.stories.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+
+export default {
+  title: 'Components/ToggleSwitch',
+  parameters: {
+    layout: 'padded'
+  },
+  excludeStories: ['ToggleSwitchTemplate'],
+  controls: { expanded: true },
+  argTypes: {
+    checked: {
+      control: {type: 'boolean'},
+      description: 'checkbox state'
+    },
+    disabled: {
+      description: 'disabled field',
+      control: {type: 'boolean'}
+    },
+    size: {
+      options: ['medium', 'small'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'size'
+    },
+    labelPosition: {
+      options: ['start', 'end'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'label position'
+    }
+  }
+}
+
+function classNamesForSwitch(disabled, checked, size, labelPosition) {
+  const classNames = ['ToggleSwitch'];
+
+  if (checked) {
+    classNames.push("ToggleSwitch--checked")
+  }
+  if (disabled) {
+    classNames.push("ToggleSwitch--disabled")
+  }
+  if (size === 'small') {
+    classNames.push("ToggleSwitch--small")
+  }
+  if (labelPosition === 'end') {
+    classNames.push('flex-row-reverse')
+  } else {
+    classNames.push('flex-row')
+  }
+
+  return classNames.join(' ')
+}
+
+export const ToggleSwitchTemplate = ({disabled, checked, size, labelPosition}) => (
+  <>
+    <toggle-switch class={classNamesForSwitch(disabled, checked, size, labelPosition)}>
+      <span aria-hidden="true" className="ToggleSwitch-labels">
+        <div className="text-right ToggleSwitch-onLabel" style={{visibility: checked ? 'visible' : 'hidden' }}>On</div>
+        <div className="text-right ToggleSwitch-offLabel" style={{visibility: checked ? 'hidden' : 'visible' }}>Off</div>
```

Comment:
Same here... the `text-right` could be replaced by `.ToggleSwitch-labels { text-align: right; }`.

Reply:
For sure, nice catch 😄